### PR TITLE
fix: balanced-brackets-tests

### LINKED ISSUE #36774: Incorrect text for test cases in Balanced Brackets question
Many of the test cases on the balanced brackets challenge state incorrect expectations. For example `isBalanced("][")should return true.`. There are many similar test cases like this.

The tests themselves seem to be correct despite the html text. It is worth noting however, that the test cases are missing at least one case that should be added. Please consider adding `isBalanced("[]]]")should return false.`  

This case covers instances where:
1. A stack approach is used to solve the problem.
2. The programmer checks for an even number of characters before iterating.  
3. The programmer does not check for an empty stack before calling `.pop()`

Example student code where the current tests pass but this case would fail:
```js
function isBalanced(str) {
    let matched = [];
    if (str.length == 0) {
        return true;
    } else if (str.length % 2 != 0) {
        return false;
    } else {
        for (let i = 0; i < str.length; i++) {
            if (str[i] == "[") {
                matched.push(str[i]);
            } else {
                matched.pop();
            }
        }
        if (matched.length !== 0) {
            return false;
        }
        return true;
    }
}
```

Page with problem: https://learn.freecodecamp.org/coding-interview-prep/rosetta-code/balanced-brackets/

**Tell us about your browser and operating system:**
* Browser Name: Chrome 
* Browser Version: 76.0.3809.100 (Official Build) (64-bit)
* Operating System: MacOS 10.13.6

### PR DESCRIPTION
**Pull request by moT01**
I ended up just replacing the second to last test with the suggestion. How does this look @James-Crean ?

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #36774

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@moT01, just checked this out locally an everything LGTM. The updated test fails as expected using the sample code from #36774. All tests pass after changing the code slightly to check whether the stack is empty before using `.pop()`: 

```js
function isBalanced(str) {
    let matched = [];
    if (str.length == 0) {
        return true;
    } else if (str.length % 2 != 0) {
        return false;
    } else {
        for (let i = 0; i < str.length; i++) {
            if (str[i] == "[") {
                matched.push(str[i]);
            } else {
                if (matched.length > 0) {
                    matched.pop();
                } else {
                    return false;
                }
            }
        }
        return true;
    }
}
```

Will wait for another review before merging.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**
